include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -118,32 +118,32 @@ class LAPACKFullMatrix : public TransposeTable<number>
   /**
    * This operator assigns a scalar to a matrix. To avoid confusion with
    * constructors, zero (when cast to the @p number type) is the only
-   * value allowed for <tt>d</tt>
+   * value allowed for @p d.
    */
   LAPACKFullMatrix<number> &
   operator = (const number d);
 
   /**
-   * This operator multiplies all entries by a fixed factor.
+   * This operator multiplies all entries by a fixed @p factor.
    */
   LAPACKFullMatrix<number> &
   operator*= (const number factor);
 
   /**
-   * This operator divides all entries by a fixed factor.
+   * This operator divides all entries by a fixed @p factor.
    */
   LAPACKFullMatrix<number> &
   operator/= (const number factor);
 
   /**
-   * Simple addition of a scaled matrix, i.e. <tt>*this += a*A</tt>.
+   * Simple addition of a scaled matrix, i.e. $\mathbf A \mathrel{+}= a \, \mathbf B$.
```

Comment:
A question before I go on with the rest of the PR: Do we use the `\mathbf{A}` with matrix A regularly within deal.II? I don't have the time to look a representative share of our code base up now, but a quick grep suggests that we mostly use it for vectors and for matrices in only a few places. I do not have a strong opinion on this - as I've been long enough in an engineering context where the change suggested here is common, but still are a mathematician inside who is somewhat reluctant to decorate the matrices beyond the usual uppercase letters.

Reply:
we use both, I agree that mostly we don't add `\mathbf`, I am fine dropping this and sticking with linear algebra notation without boldness and extra `\cdot`. 

My main point was to use LaTeX consistently.

Reply:
Although I am biased in the sense that I use the bold notation myself, I do feel that it serves a purpose and contributes positively to the understanding of the documented functions. Within a single instance of the various linear algebra classes we sometimes refer to matrices/tensors, vectors and scalars. With the current style one cannot immediately distinguish between them (e.g. lower case used [here](https://dealii.org/developer/doxygen/deal.II/classLAPACKFullMatrix.html#ab4ca715eea6ddcaba845528d698639fd) for scalars vs [here](https://dealii.org/developer/doxygen/deal.II/classLAPACKFullMatrix.html#ac3d50458d4044c0b2e4905ada88675cf) for vectors), or we use a second notation to circumvent the issue (e.g. [here](https://dealii.org/developer/doxygen/deal.II/classTrilinosWrappers_1_1MPI_1_1Vector.html#aeea1dd6795724b992526a6355612dda1) where the vector is now represented by a capital). I understand that they are contextually differentiable from one another, but we are inconsistent. I think that it would be good if we at least tried to adopt some clear, clean and consistent notation throughout the documentation.

Reply:
I would definitely agree on trying to have some uniform notation concept - but I won't be a big help in decision making because I'm not clear myself which one would be preferable.

Reply:
github should really have mechanism to do polls to make such things easier...